deploy/Vagrantfile

```diff
@@ -7,6 +7,8 @@ Vagrant.configure("2") do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
   config.vm.synced_folder ".", "/home/vagrant/sync", disabled: true
+  config.vm.network "forwarded_port", guest: 9090, host: 30600
+  config.vm.network "forwarded_port", guest: 9000, host: 30800  
```

Comment:
I think these lines are going to expose grafana and prometheus to the world, not just my laptop. Can we add `host_ip: 127.0.0.1` to these or does it break something?

Reply:
Your'e right. The forwarded port is going to bind to all interfaces, so it will be visible to all the devices in the same network. Will use ```host_ip:127.0.01``` to bind to localhost.